src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -796,7 +796,9 @@ def _exec(
             root_run_id=run_id,
             run_id=line_run_id,
             parent_run_id=run_id,
-            inputs={k: inputs[k] for k in self._flow.inputs if k in inputs},
+            # Assign inputs after validation and load_multimedia_data,
+            # incase of input is wrong and can't be persisted by storage.
+            inputs=None,
```

Comment:
inputs is None by default. I think we can just remove this line (along with the newly added comments) to avoid confusing.

Reply:
Just write `None` and comments to indicate it's by design to not set inputs here.
Since there is `inputs` in parameter
![image](https://github.com/microsoft/promptflow/assets/17527303/fc81a2c2-b696-4365-aaaf-b512db1017e1)